odoo/modules/loading.py

```diff
@@ -246,6 +246,19 @@ def load_module_graph(cr, graph, status=None, perform_checks=True,
             # update made to the schema or data so the tests can run
             # (separately in their own transaction)
             cr.commit()
+            regular_models = [model for model in model_names if not registry[model]._abstract]
+            if regular_models:
+                cr.execute("""select model from ir_model where id NOT IN (select distinct model_id from ir_model_access) AND model IN %s""", (tuple(regular_models),))
```

Comment:
I suggest to pass parameters as a list, as it is more readable than tuples.
I also suggest to uppercase SQL keywords, and put `model IN %s` before `id IN ...`